org.civicrm.reqangular/js/src/common/services/api.js

```diff
@@ -140,7 +140,7 @@ define([
       /**
        * Sends a GET request to the backend endpoint
        *
-       * @param  {String} entity - The entity the request is asking for (Contact, Appraisal, etc)
+       * @param  {String} entity - The entity the request is asking for (Contact, etc)
```

Comment:
this comments does not make sense anymore, please change to smth like:

`The entity the request is asking for (e.g. Contact)`

also please check other similar places